### ISSUE TITLE
gpu not responding when extracting facets

### ISSUE THREAD

**Issue by tomavisar** [iperov/DeepFaceLab#5268]
THIS IS NOT TECH SUPPORT FOR NEWBIE FAKERS
POST ONLY ISSUES RELATED TO BUGS OR CODE

## Expected behavior

trying to use data_src facest extract.bat, would expect it to find faces as it did with my older gpu (gtx 1080)

## Actual behavior

takes a long time displaying the 'extracting faces...' message, then shows the progress bar which dosent go up and after a few minutes it displays this message '

## Steps to reproduce

only thing i have changed is the graphics card. had a gtx1080, now have a 3090.

## Other relevant information
windows version 10.0.19042

system specs:
5950x 16 core processor 
32gb ddr4 3600 ram
rtx 3090

**Comment by tomavisar** [comment]
also using a riser cable but it seems to be working fine for everything else

**Comment by GuusDeKroon** [comment]
Yup, the same happens for me. I've tried all documentation I could find. I'm using the newest version of windows with GPU scheduling, and updated all dependencies. I'm now running the extraction on a seperate machine.

**Comment by pierrelb1** [comment]
I have the same problem with my 3080 with the latest nvidia DFL build. I tried reinstalling tensorflow and cuda11.1... Did you find a way to make it work ?

**Comment by kingkeikaku** [comment]
I am having the exact same issue. Using 2.0 . I tried DDU with full driver updates for GPU, complete clean wipe and reinstallation of windows, and nothing has helped. Ran 3d Mark to test the GPU was okay and it worked fine, have my HDMI plugged directly into GPU port and everything and it works fine. I'm not sure what the issue is but it seems to be deepfacelab specific.
Weird thing is like 3 weeks ago my GE Force RTX 3090 worked just fine.

You CAN in the meantime have your facesets extracted with CPU, but that turns what my GPU could do in 1 hour to a 36 hour task which is not going to work for me personally.  Very interested if anyone has found any solutions?

**Comment by kingkeikaku** [comment]
> I have the same problem with my 3080 with the latest nvidia DFL build. I tried reinstalling tensorflow and cuda11.1... Did you find a way to make it work ?

Did reinstalling tensor/cuda work for you?

**Comment by pierrelb1** [comment]
> > I have the same problem with my 3080 with the latest nvidia DFL build. I tried reinstalling tensorflow and cuda11.1... Did you find a way to make it work ?
> 
> Did reinstalling tensor/cuda work for you?

No, I think it solved itself by trying multiple times and at some point the extraction just started but veeery slowly. and training was using mostly CPU (even if i selected GPU) and the GPU Vram at the same time.

**Comment by ogladkov** [comment]
Does tensorflow import in your environment well? I had that problem and found out that tf couldn't import properly. Try it manually in your env.

**Comment by jfarmer33** [comment]
did you find a resolve to this issue, i just got a nvidia rtx 3060 and im having the same issue it looks like

**Comment by young169** [comment]
> Does tensorflow import in your environment well? I had that problem and found out that tf couldn't import properly. Try it manually in your env.

My cuda version is 11.x and in tensorflow, "tf.test.is_gpu_available()" gives me a 'True'. But I cannot extract faces using 3090 GPU, just the same as [tomavisar](https://github.com/tomavisar) mentioned

**Comment by ghost** [comment]
Did you ever find the answer?

**Comment by cottonpants** [comment]
Also having this exact problem with my nvidia RTX a2000.  my deepfacelive install works fine.  I have tried tinkering with CUDA versions and tensorflow, but actually couldn't manage to install tensorflow 2.10 - it would only let me install 2.12 which isn't CUDA compatible.  
Any suggestions for GPU diagnostics from within DFL would be appreciated.
